source/assets/styles/style.css

```diff
@@ -144,4 +144,63 @@ section:not(section.shown) {
 
 #fav-cards--wrapper {
   grid-area: sidebarr;
-}
\ No newline at end of file
+
+}
+
+a:link {
+  text-decoration: none;
+}
+
+.recipe-card:hover {
+  box-shadow: 0px 0px 20px black;
+}
+
+.card-body {
+  font-family: 'Merriweather', sans-serif;
+  margin: 10px;
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(10rem, 15rem));
+  justify-content: space-evenly;
```

Comment:
check code scanning comment, but im guessing space-between might be more suitable